draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -2326,6 +2326,11 @@ Host: www.example.org
    &MUST; generate a new Host field value based on the received
    request-target rather than forward the received Host field value.
 </t>
+<t>
+   When an origin server receives a request with an absolute-form of
+   request-target, the origin server must ignore the received Host header
```

Comment:
needs to take into account absolute-form URIs that do not have an authority component, such as URNs and mailto and ...

Reply:
note that, in this case, the Host would be sent with empty value.